zebra-network/src/peer/handshake.rs

```diff
@@ -175,9 +180,29 @@ where
                 return Err(HandshakeError::UnexpectedMessage(remote_msg));
             }
 
-            // XXX here is where we would set the version to the minimum of the
-            // two versions, etc. -- actually is it possible to edit the `Codec`
-            // after using it to make a framed adapter?
+            // XXX in zcashd remote peer can only send one version message and we would disconnect here if it received a second one. Is it even possible for that to happen to us here?
```

Comment:
It's not possible for it to happen here; duplicate version messages are handled in `Connection::handle_message_as_request`.